std/json.d

```diff
@@ -34,28 +52,9 @@ Synopsis:
     // list already exists, so .object optional
     jj["list"].array ~= JSONValue("D");
 
-    s = j.toString();
-    writeln(s);
-----
-
-Copyright: Copyright Jeremie Pelletier 2008 - 2009.
-License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
-Authors:   Jeremie Pelletier, David Herberth
-References: $(LINK http://json.org/)
-Source:    $(PHOBOSSRC std/_json.d)
-*/
-/*
-         Copyright Jeremie Pelletier 2008 - 2009.
-Distributed under the Boost Software License, Version 1.0.
-   (See accompanying file LICENSE_1_0.txt or copy at
-         http://www.boost.org/LICENSE_1_0.txt)
-*/
-module std.json;
-
-import std.conv;
-import std.range.primitives;
-import std.array;
-import std.traits;
+    string jjStr = `{"language":"D","list":["a","b","c","D"],"rating":3.14}`;
+    assert(jj.toString == jjStr);
```

Comment:
[Forum discussion](http://forum.dlang.org/post/nh5e2t$15vu$1@digitalmars.com) is here - in case anyone wants to follow up.

Imho the best way to go is to fix this with the compiler/druntime and printing a better error message if the assert fails.

As stated in the forum the idea of this PR is move from text-based unittests to one that we can programatically check, in this case it wouldn't compile, because it lacks a couple of imports.
And as many people stated on the forum `assert` isn't that bad and we can maybe make it better ;-)